audformat/core/attachment.py

```diff
@@ -59,6 +60,38 @@ def __init__(
         self.path = path
         r"""Attachment path"""
 
+    @property
+    def files(
+            self,
+    ) -> typing.List[str]:
+        r"""List all files part of the attachment.
+
+        List recursively the relative path
+        of all files that exist
+        under :attr:`audformat.Attachment.path`
+        on hard disc.
+
+        """
+        files = []
+        if not self._db.root:
+            return files
+
+        path = audeer.path(self._db.root, self.path)
+        if not os.path.exists(path):
```

Comment:
Maybe we should also raise an error here.

Reply:
We now raise an error.